src/qutip_tensorflow/core/data/tftensor.py

```diff
@@ -0,0 +1,98 @@
+import tensorflow as tf
+from tensorflow.errors import InvalidArgumentError
+import qutip
+import numbers
+
+__all__ = ["TfTensor"]
+
+
+class TfTensor(qutip.core.data.Data):
+    """This class provide a wraps around TensorFlow's Tensor. It will store data
+    as a Tensor of dtype `tensorflow.complex128`. Data will be expanded into a 2D
```

Comment:
you need double back-ticks (everywhere) to make this render as inline code, as opposed to in italics.